app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -125,6 +125,21 @@ def member_error_check(members)
       @company.public_send(ScimRails.config.scim_users_scope).find(member_ids)
     end
 
+    def put_error_check
+      member_error_check(params["members"])
+        
+      return if params[:active].nil?
+
+      case params[:active]
+      when true, "true", 1
+        
+      when false, "false", 0
+        
```

Comment:
Yup, the idea is to raise an error if the active attribute of the params isn't any of `true`, `"true"`, `1`, `false`, `"false"`, or `0` but I suppose I can restructure this as well since the true and false cases do the same thing